### PR TITLE
Person details

### PR DESCRIPTION
**Pull request by WiktoriaNizielska**
I created personDetails page (new hook fetched cast and crew) and noMovie image.

### CODE REVIEW

File: src/features/people/personDetails/index.js [hunk 1]
```diff
@@ -1,5 +1,108 @@
...
+                : <NoMoviePoster />
+              }
+              <TextWrapper>
+                <Container>
+                  <Title>
+                    {movie.title}
+                  </Title>
+                  {movie.release_date ? <Year>{movie.character} ({formatYear(movie.release_date)})</Year> : null}
+                  <GenresContainer>
+
```
Comment (igi144) [inline src/features/people/personDetails/index.js:55]: Empty line in here :)

File: src/features/people/personDetails/index.js [hunk 2]
```diff
@@ -1,5 +1,108 @@
...
+                <RateContainer>
+                  <Star />
+                  <Rate>{movie.vote_average.toFixed(1)}</Rate>
+                  <Votes>{movie.vote_count} votes</Votes>
+                </RateContainer>
+              </TextWrapper>
+            </MainPageMovie>
+          )}
+        </MainPageContainer>
+
```
Comment (igi144) [inline src/features/people/personDetails/index.js:67]: Another empty line :)

File: src/features/people/personDetails/index.js [hunk 3]
```diff
@@ -1,5 +1,108 @@
...
+              }
+
+              <TextWrapper>
+                <Container>
+                  <Title>
+                    {movie.title}
+                  </Title>
+                  {movie.release_date ? <Year>{movie.job} ({formatYear(movie.release_date)})</Year> : null}
+                  <GenresContainer>
+
```
Comment (igi144) [inline src/features/people/personDetails/index.js:93]: and here

**Review by igi144 — APPROVED** [review]
Just a couple empty lines, the rest is perfect. Great job Wiktoria :)

File: src/features/people/personDetails/index.js (hunk 3 above)
Comment (WiktoriaNizielska) [reply src/features/people/personDetails/index.js:93]: I will correct it next time  :)

**Pull request merged**